README.md

```diff
@@ -39,21 +39,21 @@ The Docker installation package consists of two independent products which insta
   
 It is easy to confuse which system you are currently working with.
   
-Docker Desktop has a builtin daemon which works in parallel with the `dockerd` daemon.
+Docker Desktop has a built-in daemon which works in parallel with the `dockerd` daemon.
   
-Docker Desktop keep it's settings and storage of images and containers in the user's home directory.
+Docker Desktop keeps its settings and storage of images and containers in the user's home directory.
   
-Docker Engine keep it's settings and the storage in the system directories.
+Docker Engine keeps its settings and the storage in the system directories.
```

Comment:
"settings and storage" rather than "the storage"

Reply:
But Docker really keeps its settings and storages in different places. Docker Engine in /var/lib/docker and /etc/docker but Docker Desktop in ~/.docker

Reply:
Understood.  It says "settings and the storage".  Change it to "settings and storage", which is already the (correct) construct used in the line above.

Reply:
> Understood. It says "settings and the storage". Change it to "settings and storage", which is already the (correct) construct used in the line above.

Done